### DISCUSSION TITLE
404 errors - Next.js app deploy is ignoring /pages

### DISCUSSION THREAD

**Discussion by ericelliott** [vercel/vercel#6756] [Help] [answered]
I have a next.js project created with `create-next-app`. Pages are in the project root `/pages` directory. Everything works fine when I run the app locally with `npm run dev` or `vercel dev`, but when I push to production, I get 404 errors. Looking at the output tab on the Vercel web UI, I only see the files in the `/public` folder. The `Source` tab in the UI points to my GitHub project (which is a private repo). The server build log lists all the pages I'm expecting to see in the "Generating static pages" and page optimization output.

I've hit a dead end trying to troubleshoot myself. Any hints about what to look at next would be greatly appreciated.

**Comment by leerob** [comment]
Hey Eric! Sorry about this. We'll take a look and report back here.

**Comment by ericelliott** [comment, marked as answer]
Deleting and re-creating the project in the Vercel web UI fixed the issue.

**Reply by leerob** [reply]
Ahh, well I'm sorry you had an issue, but glad it's been resolved. We will look into this regardless 😄
